src/main/java/com/google/edith/servlets/Receipt.java

```diff
@@ -42,10 +42,44 @@ public Receipt(
     this.items = items;
   }
 
+  public String toString() {
+    return (this.userId
+        + " "
+        + this.storeName
+        + " "
+        + this.date
+        + " "
+        + this.name
+        + " "
+        + this.fileUrl
+        + " "
+        + this.totalPrice);
+  }
```

Comment:
If Receipt is an AutoValue type it will implement toString() for you

Reply:
Implementing it in a later PR